Components/com/structr/structr/2.1.4/defaults.scenario.yaml

```diff
@@ -0,0 +1,39 @@
+scenario:
+  ## Source setup
+  src:
+    # This is the scenario component name which will be automatically filled. Default is ignored but must not be empty.
+    component: .
+    # This is the cache directory for downloaded files, like structr.zip or WODA-current.tar.gz
+    cachedir: ~/.cache/ONCE.2023-Scenarios
+
+  ## Server setup
+  server:
+    # What is the server, the scenario will be deployed?
+    name: test.wo-da.de
+    # What is the SSH config the server can be connected with?
+    sshconfig: WODA.test
+    # What is the scenarios root directory on the server?
+    configsdir: /var/dev/ONCE.2023-Scenarios
+    # What is the EAMD root directory on the server?
+    eamddir: /var/dev/EAMD.ucp/
+    # Where to find the servers certificate?
+    certificatedir: /var/dev/ssl/data/certbot/conf/live/test.wo-da.de
```

Comment:
Wenn du automatisch das Script für den Renewal-Hook für den Certbot anlegen möchtest, musst du noch die Variable `certconfigdir` setzen.

# Where to find the servers letsencrypt base dir?
certconfigdir: /var/dev/ssl/data/certbot

Reply:
Reicht das, das hinzuzufügen, oder muss da noch mehr code rein?

Reply:
Sollte ausreichen. Lässt sich im scenario.deploy Script Zeile 384 nachvollziehen.

Reply:
Habe es hinzugefügt